test/tools/IsolTestOptions.cpp

```diff
@@ -63,6 +63,7 @@ IsolTestOptions::IsolTestOptions(std::string* _editor):
 		("editor", po::value<std::string>(_editor)->default_value(editorPath()), "Path to editor for opening test files.")
 		("help", po::bool_switch(&showHelp), "Show this help screen.")
 		("no-color", po::bool_switch(&noColor), "Don't use colors.")
+		("yes,y", po::bool_switch(&yesToAll), "Yes to all questions/updates.")
```

Comment:
Should this be `--update` and `-u` instead?

Reply:
Should it be just `--yes` and no `-y`?

Reply:
My vote would be for `--autoupdate` or `--accept-updates`. Isoltest does not ask a yes/no question. 

Also, `--yes` does not give you any idea of what you might be agreeing to and one might not be aware of all the questions isoltest could ask. How can I know it doesn't in some circumstances ask me if it's ok to delete some stuff I might not want deleted?

Another argument against `--yes` is that I would interpret it as something like "assume yes and don't bother me with any interactive questions" but that's not the case. If there's an exception, it still stops and asks because there's no "update" option in that case.

Reply:
I would vote for `--autoupdate` as it is what it actually does

Reply:
I think `--accept-updates` may sound better, as `--autoupdate` may imply the process is fully automatic. And in case of failures or other questions it is not.

I'm fine with either options.